akka-remote/src/main/scala/akka/remote/artery/EnvelopeBufferPool.scala

```diff
@@ -557,7 +557,7 @@ private[remote] final class EnvelopeBuffer(val byteBuffer: ByteBuffer) {
     byteBuffer.rewind()
     val bytes = new Array[Byte](byteBuffer.remaining)
     byteBuffer.get(bytes)
-    val newByteBuffer = ByteBuffer.wrap(bytes)
+    val newByteBuffer = ByteBuffer.wrap(bytes).order(ByteOrder.LITTLE_ENDIAN)
```

Comment:
What system is this failing on, AFAIK we only support artery on little endian systems where this should be the default.

Reply:
Oh right, this doesn't come from Artery so will default to Big endian